Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -326,29 +326,47 @@ void HackStudioWidget::set_edit_mode(EditMode mode)
 
 NonnullRefPtr<GUI::Menu> HackStudioWidget::create_project_tree_view_context_menu()
 {
+    m_new_plain_file_action = create_new_file_action("Plain File", "/res/icons/16x16/new.png", "");
+    m_new_cpp_file_action = create_new_file_action("C++ Source File", "/res/icons/16x16/filetype-cplusplus.png", "cpp");
+    m_new_header_file_action = create_new_file_action("C++ Header File", "/res/icons/16x16/filetype-header.png", "h");
+    m_new_javascript_file_action = create_new_file_action("JavaScript Source File", "/res/icons/16x16/filetype-javascript.png", "js");
+    m_new_html_file_action = create_new_file_action("HTML File", "/res/icons/16x16/filetype-html.png", "html");
```

Comment:
It seems a little weird to me to have HTML and JS as options but not have CSS. Though, it looks like there's no icon for that.

GML would be another good candidate, as there's a built-in GML preview widget. That's also lacking an icon though.

Reply:
I totally forgot about CSS. I thought about adding GML, but as you said we don't have an icon for that. I tried my hand at making one but it looked horrible, so I just didn't put it.
I was planning on making an issue for that so that someone more artistically gifted could try :)

Reply:
You could add them, keep the icon as a plaintext one and leave a FIXME next to it.